src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -104,13 +112,19 @@ protected boolean isFinished() {
 
     public Command driveBackwards() {
         return new Command("Drive Backwards", this) {
+            private double triggerSpeed = 0;
```

Comment:
You can make these variables right in `execute()`, they're not used anywhere else

Reply:
gotcha